src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -560,9 +639,13 @@ private static Collection<ScriptBindingProvider> LoadBindingProviders(ScriptHost
                 typeof(NotificationHubScriptBindingProvider),
                 typeof(SendGridScriptBindingProvider),
                 typeof(TwilioScriptBindingProvider),
-                typeof(BotFrameworkScriptBindingProvider)
-            };
+                typeof(BotFrameworkScriptBindingProvider),
```

Comment:
Discussed offline - we're confident the model works for new types, but it has known issues for a) triggers (byob doesn't support triggers) and b) minor inconsistencies (which wouldn't be an issue for new bindings) 
I'll apply this to a few (table, sendgrid), but we won't replace all at this time.